src/dates/date-format-placeholder.js

```diff
@@ -0,0 +1,78 @@
+import { localeInfo } from '../cldr';
+
+const MONTH = 'month';
+const DAY = 'day';
+const HOUR = 'hour';
+const ZONE = 'zone';
+const WEEKDAY = 'weekday';
+const QUARTER = 'quarter';
+
+const dateFieldMap = {
+    'y': 'year',
+    'M': MONTH,
+    'L': MONTH,
+    'd': DAY,
+    'E': DAY,
```

Comment:
I tend to agree here, although `weekday` will return smth like `day of the week`, which is a bit longer to display in the input. Nevertheless, it's semantically correct I think.
Will change it to 'weekday'.